### PR TITLE
Updating commission related forms and buttons.

### PR DESCRIPTION
**Pull request by skullbunnygalaxy**

### CODE REVIEW

File: lib/banchan_web/live/denizen_live/show.ex [hunk 1]
```diff
@@ -77,7 +77,7 @@ defmodule BanchanWeb.DenizenLive.Show do
               <LiveRedirect
                 label="Edit"
                 to={Routes.denizen_edit_path(Endpoint, :edit, @user.handle)}
-                class="text-center rounded-full py-1 px-5 bg-amber-200 text-black m-1"
+                class="text-center  btn btn-sm btn-primary m-3"
```
Comment (zkat) [inline lib/banchan_web/live/denizen_live/show.ex:80]: extra space here

File: lib/banchan_web/live/studio_live/offerings/edit.ex [hunk 2]
```diff
@@ -31,8 +31,12 @@ defmodule BanchanWeb.StudioLive.Offerings.Edit do
       current_user_member?={@current_user_member?}
       tab={:settings}
     >
-      <h1>Edit Offering</h1>
-      <Components.Offering id="edit-offering" changeset={@changeset} submit="save" />
+      <div class="card shadow bg-base-200 card-bordered text-base-content lg:card-side">
```
Comment (zkat) [inline lib/banchan_web/live/studio_live/offerings/edit.ex:34]: shouldn't we use our Card component for this?

File: lib/banchan_web/live/studio_live/offerings/new.ex [hunk 3]
```diff
@@ -30,9 +30,15 @@ defmodule BanchanWeb.StudioLive.Offerings.New do
       current_user_member?={@current_user_member?}
       tab={:settings}
     >
-      <h1>New Offering</h1>
-      <h2>Make a new offering to the gods</h2>
-      <Components.Offering id="new-offering" changeset={@changeset} submit="save" />
+      <div class="card shadow bg-base-200 card-bordered text-base-content lg:card-side">
```
Comment (zkat) [inline lib/banchan_web/live/studio_live/offerings/new.ex:33]: same as above, re: using Card

File: lib/banchan_web/live/studio_live/offerings/edit.ex (hunk 2 above)
Comment (skullbunnygalaxy) [reply lib/banchan_web/live/studio_live/offerings/edit.ex:34]: Yeah, I was using it as a placeholder to figure out spacing, but I realize now I should just use the margin class unless I actually want to make something a card.

**Review by zkat — APPROVED** [review]

**Pull request merged**